README.md

```diff
@@ -13,50 +13,120 @@
   </p>
 </div>
 
+## Release notes
+**Release notes and unreleased changes can be found in the [CHANGELOG](CHANGELOG.md)**
+
 ## Requirements
 - rust v1.56 and up
-- MacOS (x86), M1 (through rosetta) or Linux (x86) for sandbox tests. Testnet is available regardless
+- MacOS (x86) or Linux (x86) for sandbox tests. Testnet is available regardless
+
+### M1 MacOS
+NOTE: Current version of workspaces does not provide support for M1 macbooks, due to some internal upgrades with wasmer. Look at using for workspaces version `0.1.1` for now until we get this settled out or check the progress of it with this [issue](https://github.com/near/workspaces-rs/issues/110).
```

Comment:
NOTE: Current version of workspaces-rs does not support use on M1 chip devices due to internal upgrades with wasmer. M1 users should use workspaces-rs version `0.1.1` until this problem has been addressed. Check the progress with on this issue [here](https://github.com/near/workspaces-rs/issues/110).